usr/share/rear/prep/RSYNC/default/150_check_rsync_protocol_version.sh

```diff
@@ -45,6 +45,10 @@ if [ "${RSYNC_USER}" != "root" ]; then
             BACKUP_RSYNC_OPTIONS=( "${BACKUP_RSYNC_OPTIONS[@]}" --rsync-path="""rsync --fake-super""" )
         fi
     else
-        Error "rsync --fake-super not possible on system ($RSYNC_HOST) (please upgrade rsync to 3.x)"
+        if [ ${BACKUP_RSYNC_OPTIONS[@]/--fake-super/} != ${BACKUP_RSUNC_OPTIONS[@]} ]; then
+            Error "rsync --fake-super not possible on system ($RSYNC_HOST) (please upgrade rsync to 3.x)"
+        else
+            Log "Warning: rsync --fake-super not possible on system ($RSYNC_HOST) (please upgrade rsync to 3.x)"
+        fi
```

Comment:
I suppose the intent was to not abort in the case when the rsync daemon is used, where we assume that version is 29. This whole part could never work properly though. There is a typo, the use of `[@]` is wrong (you need `[*]` with quotes to prevent word splitting) and the intent is wrong: if the protocol is 29, we should abort unconditionally, because we need fake super, even if not specified in `BACKUP_RSYNC_OPTIONS`. `--rsync-path="rsync --fake-super"` will not work for tthe rsync daemon connection anyway.